pages/index.tsx

```diff
@@ -21,7 +30,18 @@ const Home: FC<InlineResponse200> = ({ postsCount }): ReactElement => {
 };
 
 export const getServerSideProps: GetServerSideProps = async () => {
-  const { data } = await getPosts();
+  const { message, response } = (await getPosts()) as IErrorHandle.IErrorData;
```

Comment:
WHAT IS THIS!

Reply:
here i'm destructing `message` and `response` if we have an error and passing them to `<Error  />` component that imported from `'next/error'`

Reply:
where did you read this approach?

Reply:
no where, there wasn't many resources for error handling in `getServerSideProps`, i want more explanation for what is going on wrong and i will fix it

Reply:
I'll join the stand up. let's talk there.